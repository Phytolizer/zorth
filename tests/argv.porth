include "std.porth"

2drop
while dup 0 > do
  dup strlen swap
  stdout write drop
  "\n" stdout write drop
end
