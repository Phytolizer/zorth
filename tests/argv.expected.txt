: returncode 0
: stdout 0

: stderr 0

